doc/user_guide/tools.rst

```diff
@@ -10,36 +10,43 @@ The Signal class and its subclasses
     Do not worry if you do not understand it all.
 
 
-HyperSpy stores the data in the :py:class:`~.signal.Signal` class, that is
+HyperSpy stores the data in the :py:class:`~.signal.BaseSignal` class, that is
 the object that you get when e.g. you load a single file using
 :py:func:`~.io.load`. Most of the data analysis functions are also contained in
-this class or its specialized subclasses. The :py:class:`~.signal.Signal` class
+this class or its specialized subclasses. The :py:class:`~.signal.BaseSignal` class
 contains general functionality that is available to all the subclasses. The
 subclasses provide functionality that is normally specific to a particular type
-of data, e.g. the :py:class:`~._signals.spectrum.Spectrum` class provides common
-functionality to deal with spectral data and
+of data, e.g. the :py:class:`~._signals.signal1D.Signal1D` class provides common
+functionality to deal with one-dimensional (e.g. spectral) data and
 :py:class:`~._signals.eels.EELSSpectrum` (which is a subclass of
-:py:class:`~._signals.spectrum.Spectrum`) adds extra functionality to the
-:py:class:`~._signals.spectrum.Spectrum` class for electron energy-loss
+:py:class:`~._signals.signal1D.Signal1D`) adds extra functionality to the
+:py:class:`~._signals.signal1D.Signal1D` class for electron energy-loss
 spectroscopy data analysis.
 
+.. versionchanged:: 0.8.5
+
 Currently the following signal subclasses are available:
 
-* :py:class:`~._signals.spectrum.Spectrum`
-* :py:class:`~._signals.image.Image`
+* :py:class:`~._signals.signal1D.Signal1D`
```

Comment:
After making the name of the files lower-case, the module names need updating here (there) and everywhere e.g. `~._signals.signal1D.Signal1D` -> `~._signals.signal1d.Signal1D`